force-app/main/default/classes/CheckFulfillmentOrders_Batch.cls

```diff
@@ -0,0 +1,39 @@
+public class CheckFulfillmentOrders_Batch Implements Database.Batchable<sObject>, Database.Stateful {
+
+    public Database.QueryLocator start(Database.BatchableContext bc) {
+        FO_immediate_attention_delay__mdt fOImmediateAttentionDelayMdt = 
+            [SELECT Number_Of_Hours__c FROM FO_immediate_attention_delay__mdt];
+
+        datetime searchDate = datetime.now().addHours(-fOImmediateAttentionDelayMdt.Number_Of_Hours__c.intValue());
+
+        List<Order> orders = [SELECT Id FROM Order WHERE OrderedDate < :searchDate];
```

Comment:
You don't need to query Orders to filter their orderedDate, in the FulfillmentOrder query you can filter it with WHERE OrderSummary.OrderedDate < :date for example, so this list is not necessary